docs/fragments/pr-sanitizer.adoc

```diff
@@ -33,9 +33,25 @@ include::../../pkg/plugin/pr-sanitizer/test_fixtures/github_calls/pr-sanitizer.y
 <1> Defines set of custom types which will be used to match titles prefixes in changeset and determine if Pull Requests Title conforms with the semantic message style. (Eg. above code snippet configures link:https://medium.com/walmartlabs/semantic-commit-messages-with-emojis-dba2541cea9a[emoji style] semantic commit message.)
 <2> Allows you to decide if you want to combine your patterns with the list (`feat`, `fix`, `refactor`, `docs`, `test`, `chore`, `style`) of predefined default types (`true` by default).
 
+
+=== Description - Content Length Verification [[description-content-length-verification]]
+
+This ensures that the PR description has more than 50 characters to have more elaborated description from which anyone can understand all changes proposed in this PR.
+
+All issue link keywords(e.g. `close #1`) added to close an issue are not taken into consideration while measuring description length.
+
+If it won't find Pull Request description more than 50 characters, it adds a comment with a hint message and marks its check as **Failure**, or as **Success** otherwise.
+
+=== Description - Issue Link Verification [[description-issue-link-verification]]
+
+This ensures that the PR description has issue link using keywords to close issue after merging pull request. Supported keywords for issue links are `closes`, `closed`, `fix`, `fixes`, `fixed`, `resolve`, `resolves`, `resolved`.
+For More info about how can you close issues via PR can be found link:https://blog.github.com/2013-05-14-closing-issues-via-pull-requests[here].
```

Comment:
I would rather point to the description in documentation https://help.github.com/articles/closing-issues-using-keywords/